cli/cluster/tarantool_test.go

```diff
@@ -1,57 +1,62 @@
 package cluster_test
 
 import (
-	"fmt"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 
+	"github.com/tarantool/go-tarantool"
 	"github.com/tarantool/tt/cli/cluster"
-	"github.com/tarantool/tt/cli/connector"
+	"github.com/tarantool/tt/cli/integrity"
 )
 
-type MockEvaler struct {
-	connector.Evaler
-	Expr string
-	Args []any
-	Opts connector.RequestOpts
-	Ret  []any
-	Err  error
+// DO NOT REMOVE the commented-out unit tests below.
```

Comment:
Please, create an issue in `tarantool/tt` with the source commit and remove the tests. We could fetch them not from the source code, but from the git history.

Reply:
Please, add the label `blocked` to the issue and link to go-tarantool `v2` upgrade issue in tt-ee.
Please, add a link to a commit with old tests before they have been removed.

Reply:
Done

Reply:
Thank you!